src/encoded/static/components/browse/components/CustomColumnController.js

```diff
@@ -97,50 +98,38 @@ export class CustomColumnSelector extends React.Component {
     }
 
     columnDefinitionsWithHiddenState(){
```

Comment:
Oh, this doesn't filter out visible columns. It adds a "hiddenState" field to each column definition. I thought this function was a filter. Can you add a function comment to clarify that?

Reply:
Oops, I merged before seeing comments!
Will do on this branch.

Reply:
Thanks!